python/lsst/pipe/analysis/fakesAnalysis.py

```diff
@@ -1199,8 +1203,15 @@ def fakesMagnitudeCompare(inputFakesMatched, processedFakesMatched, plotInfoDict
     yLabel = "Output - Input Magnitude (mmag)"
     title = "Magnitude Difference For Fake Stars \n (" + magCol + ")"
 
-    fig = plotWithOneHist(xs, ys, maskForStats, xLabel, yLabel, title, plotInfoDict)
-
+    fig, med, sigmaMad = plotWithOneHist(xs, ys, maskForStats, xLabel, yLabel, title, plotInfoDict)
+    if verifyJob:
+        magName = magCol.replace("Flux", "")
+        if "Aperture" in magCol:
+            magName = magName.replace("Aperture", "Aper")
+        if "Circular" in magCol:
+            magName = magName.replace("Circular", "Circ")
+        addMetricMeasurement(verifyJob, "pipe_analysis.fake_stars_magDiff_" + magName, med*u.mmag)
+        addMetricMeasurement(verifyJob, "pipe_analysis.fake_stars_magDiffSigMad_" + magName, sigmaMad*u.mmag)
```

Comment:
For a next iteration on this I would suggest to not encode information in the metric name and use metric specifications instead.